content/zh-cn/docs/concepts/security/pod-security-policy.md

```diff
@@ -620,9 +622,9 @@ kubectl-user get pod pause -o yaml | grep kubernetes.io/psp
 ```
 
 <!--
-The output is similar to this:
+The output is similar to this
 -->
-输出类似于：
+输出类似于
```

Comment:
```suggestion
输出类似于：
```
The original colon `：` is better. It is not necessary to follow the en here in my opinion.

Reply:
I think if the upstream is not good and cannot express the correct meaning, it should be changed upstream.

If each downstream is translated with a special case, the contributors and reviewers will have no basis to become their own creations, which will make it difficult for subsequent people to maintain.

Reply:
Adding '：' is a good suggestion, I think.